third_party/jsonnet/lib/formatter/jsonnetfmt.go

```diff
@@ -152,7 +152,14 @@ func Format(filename string, input string, options Options) (string, error) {
 	if err != nil {
 		return "", err
 	}
+	return formatAst(node, finalFodder, options)
+}
+
+func FormatAst(node ast.Node, options Options) (string, error) {
+	return formatAst(node, nil, options)
+}
 
+func formatAst(node ast.Node, finalFodder ast.Fodder, options Options) (string, error) {
```

Comment:
A to je tych par riadkov, pre ktore to robime :)

Stacilo by, aby to dali autory kniznice public.
Ale generovanie JsonNet asi nie je uplne bezny use-case.

Reply:
😆👍